TemplePlus/condition.cpp

```diff
@@ -476,6 +476,23 @@ class ConditionFunctionReplacement : public TempleFix {
 		redirectCall(0x100FC18C, AnimalCompanionLevelHook);
 		redirectCall(0x100FC2D6, AnimalCompanionLevelHook);
 		redirectCall(0x100FC6D4, AnimalCompanionLevelHook);
+
+		// fix for negative str mod still accounting in the bon list
+		static int(*orgWeaponFinesseToHitBonus)(DispatcherCallbackArgs) =
```

Comment:
One of my life's biggest regrets is this long-ass file :)
Please put it in ability_fixes.cpp